ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -0,0 +1,47 @@
+from typing import Callable, TypeVar, Union
+from durabletask import task
+from datetime import datetime
+from dapr.ext.workflow.workflow_context import WorkflowContext, Workflow
+
+from dapr.ext.workflow.workflow_activity_context import WorkflowActivityContext
+
+T = TypeVar('T')
+TInput = TypeVar('TInput')
+TOutput = TypeVar('TOutput')
+
+
+class DaprWorkflowContext(WorkflowContext):
+    """DaprWorkflowContext that provides proxy access to internal OrchestrationContext instance."""
+
+    __ignore__ = "class mro new init setattr getattr getattribute"
+
+    def __init__(self, obj: task.OrchestrationContext):
+        self._obj = obj
+
+    # provide proxy access to regular attributes of wrapped object
+    def __getattr__(self, name):
+        return getattr(self._obj, name)
+
+    def instance_id(self) -> str:
+        return self._obj.instance_id
+
+    def current_utc_datetime(self) -> datetime:
+        return self._obj.current_utc_datetime
+
+    def is_replaying(self) -> bool:
```

Comment:
Let's make `is_replaying`, `current_utc_datetime`, and `instance_id` into properties by adding the `@property` annotation to each of those methods.

Reply:
Ah, looks like you did this already in `workflow_context.py`. Maybe we don't need to do it again here.

Reply:
Provided `@property` here as well.